city_scrapers/spiders/losca_Homeless_Services.py

```diff
@@ -0,0 +1,114 @@
+from datetime import datetime
+from urllib.parse import urljoin
+
+from city_scrapers_core.constants import (
+    BOARD,
+    CITY_COUNCIL,
+    COMMISSION,
+    COMMITTEE,
+    NOT_CLASSIFIED,
+)
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class LoscaHomelessServicesSpider(CityScrapersSpider):
+    name = "losca_Homeless_Services"
+    agency = "Los Angeles Homeless Services Authority"
+    timezone = "America/Los_Angeles"
+    start_urls = ["https://www.lahsa.org/events"]
+
+    def parse(self, response):
+        """
+        Parse upcoming meetings for LAHSA.
+        """
+        for item in response.css(".col-lg-8 .list-group a"):
+            # parse classification first
+            # we need it for location
+            classification = self._parse_classification(item)
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description="",
+                classification=classification,
+                start=self._parse_start(item),
+                end=self._parse_end(item),
```

Comment:
You can just assign None to `end` and remove the `_parse_end` method on line 80.
```suggestion
                end=None,
```